cardinal/system/t_tokeniser_system_test.go

```diff
@@ -0,0 +1,81 @@
+package system
+
+import (
+	"testing"
+
+	"github.com/ArchetypalTech/TheOrugginTrail-ArgusWE/cardinal/enums"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewTokeniserSystem(t *testing.T) {
+	ts := NewTokeniserSystem()
+
+	assert.NotNil(t, ts.cmdLookup)
```

Comment:
these SHOULD return nil if no key is passed to them or rather a "None" type

Reply:
This func test the setup only, the next function test if no key or key is passed. Also adjusted the return functions on the tokeniser system so that they return a "None" type if no key is passed.